light-client/Cargo.toml

```diff
@@ -5,13 +5,14 @@ members = [
   "./ibc-client-starknet-cw",
   "./ibc-client-starknet-types",
   "./starknet-storage-verifier",
+  "./poseidon",
 ]
 
 [workspace.package]
 version      = "0.1.0"
 license      = "MIT OR Apache-2.0"
 edition      = "2021"
-rust-version = "1.72.1"
+rust-version = "1.80.0"
```

Comment:
why did you need to bump this ?

Reply:
We use a unstable call in `poseidon` which is stable with Rust v1.80:

```
warning: current MSRV (Minimum Supported Rust Version) is `1.72.1` but this item is stable since `1.80.0`
  --> poseidon/src/hades.rs:13:5
   |
13 |     LazyLock::new(|| {
```